MC_1.10.2/build.gradle

```diff
@@ -0,0 +1,62 @@
+
+// For those who want the bleeding edge
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter()
+        maven {
+            name = "forge"
+            url = "http://files.minecraftforge.net/maven"
+        }
+        maven {
+            name = "sonatype"
+            url = "https://oss.sonatype.org/content/repositories/snapshots/"
+        }
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+        flatDir {
+            dirs 'libs'
+        }
+    }
+    dependencies {
```

Comment:
As a note these grade files need to be cleaned up a lot as with my previous commit most of this is not needed at most we just need a from parent project pull all classes and resources we won't be gathering any external deps and won't be shading anything